src/test/daml/Daml/Finance/Asset/Test/Util/Holding.daml

```diff
@@ -1,24 +1,17 @@
 -- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
For my understanding, why do we need this language extension? Is it to enable the `forall a` below?

Reply:
thought it was as you said, but actually not needed here